catalog/resource_external_location.go

```diff
@@ -49,6 +49,7 @@ func ResourceExternalLocation() *schema.Resource {
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
 			return m
 		})
+	update := updateFunctionFactory("external-location", []string{"owner", "name", "comment", "url", "credential_name", "skip_validation"})
```

Comment:
with these many fields - I'll eventually move this up to common layer. thanks for making the first step! :)